specification/resources/kubernetes/responses/status_messages.yml

```
@@ -0,0 +1,16 @@
+description: |
+  The response is a JSON object which contains status messages for a Kubernetes cluster. Each message object contains a timestamp and an indication of what issue the cluster is experiencing at a given time.
+
+
+headers:
+  ratelimit-limit:
+    $ref: '../../../shared/headers.yml#/ratelimit-limit'
+  ratelimit-remaining:
+    $ref: '../../../shared/headers.yml#/ratelimit-remaining'
+  ratelimit-reset:
+    $ref: '../../../shared/headers.yml#/ratelimit-reset'
+
+content:
+  application/json:
+    schema:
+      $ref: '../models/status_messages.yml'
```

This is an array or `messages` not a single message, right?

```suggestion
      properties:
        messages:
          type: array
          items:
            $ref: "../models/status_messages.yml"
```

---

Ah yes ty!!!